.github/workflows/test-gpu.yml

```diff
@@ -0,0 +1,43 @@
+name: Test GPU
+
+on: [pull_request]
```

Comment:
why not also push?

Reply:
GPU testing will be executed on an AWS EC2 node and incur some amount of charges. I set PR-only tests for now while debugging instance spinups.

Reply:
@dzenanz @thewtex do you agree that it is reasonable to leave GPU test execution for PRs only?

Reply:
Yes, that is fine.